cfgov/v1/migrations/0006_update_tables.py

```diff
@@ -26,7 +26,7 @@ def update_content_tables(wagtail_page):
         if item['type'] == 'table':
             item['type'] = 'table_block'
             item['value'] = create_tableblock_data(old_table=item['value'])
-            print "Creating a new TableBlock on page %s" %(wagtail_page.title)
+            print "Creating a new TableBlock on page %s of page type %s" %(wagtail_page.title, wagtail_page.__class__.__name__)
 
 
 def update_full_width_text_tables(wagtail_page):
```

Comment:
Have each nested layer call a method so the for loops only go 1 deep. You can also move the if conditions to the right of the for loop on the same line.